DYDX-Bots/Perpetual-Large-Deposits-Withdrawals/README.md

```diff
@@ -0,0 +1,85 @@
+# Large deposits withdrawals on perpetual exchange
+
+## Description
+
+This bot detects deposits and withdrawals on dYdX perpetual exchange contract with large transfered amounts.
+
+> The bot can operate in two different modes, used to determine the threshold of a _large_ transfered amount.
+>
+> - `STATIC` mode refers to the bot using a static predefined threshold.
+> - `PERCENTAGE` refers to setting the threshold as a percentage of the contract balance.
+
+> In order to switch between the two modes, change `DYNAMIC_CONFIG` to `STATIC_CONFG` in agent.ts.
```

Comment:
Could specify exactly where to switch between modes. E.g. "on L62" or "the arguments in `provideHandleTransaction`," etc.